railway fail

@claude railway failed to deploy. here are the logs: 
build log: 
2026-04-25T04:02:37.516957237Z [inf]  
2026-04-25T04:02:38.189215506Z [inf]  [35m[Region: us-west1][0m
2026-04-25T04:02:38.209341881Z [inf]  
2026-04-25T04:02:38.209394625Z [inf]  [38;2;125;86;243m╭─────────────────╮[0m
2026-04-25T04:02:38.209400067Z [inf]  [38;2;125;86;243m│[0m Railpack 0.23.0 [38;2;125;86;243m│[0m
2026-04-25T04:02:38.209404619Z [inf]  [38;2;125;86;243m╰─────────────────╯[0m
2026-04-25T04:02:38.209411325Z [inf]  
2026-04-25T04:02:38.209415684Z [inf]  [97m↳ Detected Shell[0m
2026-04-25T04:02:38.209419550Z [inf]  [97m↳ Using shell script: start.sh with interpreter: bash[0m
2026-04-25T04:02:38.209424030Z [inf]  [97m↳ Found web command in Procfile[0m
2026-04-25T04:02:38.209428199Z [inf]  
2026-04-25T04:02:38.209432775Z [inf]  [1mSteps[0m
2026-04-25T04:02:38.209437215Z [inf]  [38;5;238m──────────[0m
2026-04-25T04:02:38.209441483Z [inf]  [95m▸ build[0m
2026-04-25T04:02:38.209445611Z [inf]  [38;5;245m$[0m [1msh build.sh[0m
2026-04-25T04:02:38.209449700Z [inf]  
2026-04-25T04:02:38.209453725Z [inf]  [1mDeploy[0m
2026-04-25T04:02:38.209458521Z [inf]  [38;5;238m──────────[0m
2026-04-25T04:02:38.209462902Z [inf]  [38;5;245m$[0m [1mbash start.sh[0m
2026-04-25T04:02:38.209467064Z [inf]  
2026-04-25T04:02:38.209471055Z [inf]  
2026-04-25T04:02:38.210905162Z [inf]  Successfully prepared Railpack plan for build
2026-04-25T04:02:38.210965138Z [inf]  
2026-04-25T04:02:38.210977509Z [inf]  
2026-04-25T04:02:38.211763455Z [inf]  context: nwbb-sfI2
2026-04-25T04:02:38.387806296Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-25T04:02:38.387879922Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-25T04:02:38.387902928Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-25T04:02:38.400725940Z [inf]  [internal] load build definition from railpack-plan.json
2026-04-25T04:02:38.403153034Z [inf]  resolve image config for docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0
2026-04-25T04:02:38.669129610Z [inf]  resolve image config for docker-image://ghcr.io/railwayapp/railpack-frontend:v0.23.0
2026-04-25T04:02:38.771826028Z [inf]  load build definition from railpack-plan.json
2026-04-25T04:02:38.771908608Z [inf]  load build definition from railpack-plan.json
2026-04-25T04:02:38.772392108Z [inf]  load build definition from railpack-plan.json
2026-04-25T04:02:38.783549168Z [inf]  load build definition from railpack-plan.json
2026-04-25T04:02:38.993342453Z [inf]  create mise config
2026-04-25T04:02:38.993438803Z [inf]  copy /app
2026-04-25T04:02:38.993454299Z [inf]  sh build.sh
2026-04-25T04:02:38.993463588Z [inf]  [railpack] secrets hash
2026-04-25T04:02:38.993471185Z [inf]  copy .
2026-04-25T04:02:38.993482162Z [inf]  copy / /app
2026-04-25T04:02:38.993493480Z [inf]  copy /root/.local/state/mise
2026-04-25T04:02:38.993501096Z [inf]  copy /etc/mise/config.toml
2026-04-25T04:02:38.993508656Z [inf]  copy /usr/local/bin/mise
2026-04-25T04:02:38.993516428Z [inf]  copy /mise/installs
2026-04-25T04:02:38.993527350Z [inf]  copy /mise/shims
2026-04-25T04:02:38.993535610Z [inf]  install mise packages: 
2026-04-25T04:02:38.993551051Z [inf]  [railpack] merge ghcr.io/railwayapp/railpack-runtime:mise-2026.3.17, $packages:mise, $build
2026-04-25T04:02:38.993565296Z [inf]  [railpack] secrets hash
2026-04-25T04:02:38.993582299Z [inf]  loading .
2026-04-25T04:02:38.993931272Z [inf]  docker-image://ghcr.io/railwayapp/railpack-builder:mise-2026.3.17
2026-04-25T04:02:38.993986908Z [inf]  loading .
2026-04-25T04:02:38.994001356Z [inf]  [railpack] secrets hash
2026-04-25T04:02:39.005115666Z [inf]  [railpack] secrets hash
2026-04-25T04:02:39.035791346Z [inf]  loading .
2026-04-25T04:02:39.037636633Z [inf]  copy mise.toml
2026-04-25T04:02:39.054848724Z [inf]  copy mise.toml
2026-04-25T04:02:39.056836074Z [inf]  mkdir /etc/mise
2026-04-25T04:02:39.071653962Z [inf]  mkdir /etc/mise
2026-04-25T04:02:39.073419081Z [inf]  create mise config
2026-04-25T04:02:39.087593527Z [inf]  create mise config
2026-04-25T04:02:39.089395499Z [inf]  install mise packages: 
2026-04-25T04:02:40.227633801Z [inf]  mise python@3.11.15  [1/3] install
mise node@20.20.2    [1/3] install

2026-04-25T04:02:40.229137597Z [inf]  mise node@20.20.2    [1/3] download node-v20.20.2-linux-x64.tar.gz

2026-04-25T04:02:40.364932867Z [inf]  mise python@3.11.15  [1/3] download cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-25T04:02:40.758624815Z [inf]  mise python@3.11.15  [1/3] generate checksum cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-25T04:02:40.773213932Z [inf]  mise python@3.11.15  [1/3] extract cpython-3.11.15+20260414-x86_64-unknown-linux-gnu-install_only_stripped.tar.gz

2026-04-25T04:02:41.124492747Z [inf]  mise node@20.20.2    [2/3] generate checksum node-v20.20.2-linux-x64.tar.gz

2026-04-25T04:02:41.144730405Z [inf]  mise node@20.20.2    [3/3] extract node-v20.20.2-linux-x64.tar.gz

2026-04-25T04:02:41.360319878Z [inf]  mise python@3.11.15  [1/3] python --version

2026-04-25T04:02:41.364096616Z [inf]  mise python@3.11.15  [1/3] Python 3.11.15

2026-04-25T04:02:41.373454754Z [inf]  mise python@3.11.15 ✓ installed

2026-04-25T04:02:41.961914709Z [inf]  mise node@20.20.2    [3/3] node -v

2026-04-25T04:02:41.966513427Z [inf]  mise node@20.20.2    [3/3] v20.20.2

2026-04-25T04:02:41.967354833Z [inf]  mise node@20.20.2    [3/3] npm -v

2026-04-25T04:02:42.072387712Z [inf]  mise node@20.20.2    [3/3] 10.8.2

2026-04-25T04:02:42.089937653Z [inf]  mise node@20.20.2  ✓ installed

2026-04-25T04:02:42.295785726Z [inf]  install mise packages: 
2026-04-25T04:02:42.297678750Z [inf]  copy / /app
2026-04-25T04:02:42.394949833Z [inf]  copy / /app
2026-04-25T04:02:42.397449761Z [inf]  copy .
2026-04-25T04:02:42.427242856Z [inf]  copy .
2026-04-25T04:02:42.428910143Z [inf]  sh build.sh
2026-04-25T04:02:42.533670245Z [inf]  Building frontend...

2026-04-25T04:02:42.765056986Z [inf]  copy /mise/shims
2026-04-25T04:02:42.773649018Z [inf]  copy /mise/shims
2026-04-25T04:02:44.002915180Z [inf]  copy /mise/installs
2026-04-25T04:02:44.944127132Z [inf]  npm warn deprecated whatwg-encoding@3.1.1: Use @exodus/bytes instead for a more spec-conformant and faster implementation

2026-04-25T04:02:45.347746334Z [inf]  copy /mise/installs
2026-04-25T04:02:45.347817129Z [inf]  copy /usr/local/bin/mise
2026-04-25T04:02:45.404457359Z [inf]  npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported

2026-04-25T04:02:45.572456446Z [inf]  copy /usr/local/bin/mise
2026-04-25T04:02:45.575393482Z [inf]  copy /etc/mise/config.toml
2026-04-25T04:02:45.605776794Z [inf]  copy /etc/mise/config.toml
2026-04-25T04:02:45.605876793Z [inf]  copy /root/.local/state/mise
2026-04-25T04:02:45.627202133Z [inf]  copy /root/.local/state/mise
2026-04-25T04:02:46.113341908Z [inf]  npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.

2026-04-25T04:02:46.224001638Z [inf]  npm warn deprecated glob@7.2.3: Old versions of glob are not supported, and contain widely publicized security vulnerabilities, which have been fixed in the current version. Please update. Support for old versions may be purchased (at exorbitant rates) by contacting i@izs.me

2026-04-25T04:02:46.910020335Z [inf]  npm warn deprecated @humanwhocodes/config-array@0.13.0: Use @eslint/config-array instead

2026-04-25T04:02:46.945723862Z [inf]  npm warn deprecated @humanwhocodes/object-schema@2.0.3: Use @eslint/object-schema instead

2026-04-25T04:02:49.461319274Z [inf]  npm warn deprecated eslint@8.57.1: This version is no longer supported. Please see https://eslint.org/version-support for other options.

2026-04-25T04:02:50.909800601Z [inf]  
added 472 packages, and audited 473 packages in 8s

2026-04-25T04:02:50.909827871Z [inf]  
158 packages are looking for funding

2026-04-25T04:02:50.909884732Z [inf]    run `npm fund` for details

2026-04-25T04:02:50.932812728Z [inf]  
5 moderate severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

To address all issues (including breaking changes), run:
  npm audit fix --force

Run `npm audit` for details.

2026-04-25T04:02:50.934029224Z [inf]  npm notice
npm notice New major version of npm available! 10.8.2 -> 11.13.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v11.13.0
npm notice To update run: npm install -g npm@11.13.0
npm notice

2026-04-25T04:02:51.140462411Z [inf]  
> swiftshift@0.0.0 build
> tsc && vite build


2026-04-25T04:02:57.669165948Z [inf]  src/App.tsx(1982,9): error TS6133: 'cycleTheme' is declared but its value is never read.

2026-04-25T04:02:57.669348901Z [inf]  src/App.tsx(1992,9): error TS6133: 'themeLabel' is declared but its value is never read.

2026-04-25T04:02:57.669473646Z [inf]  src/App.tsx(1993,9): error TS6133: 'themeDotColor' is declared but its value is never read.

2026-04-25T04:02:57.669489873Z [inf]  src/App.tsx(5149,11): error TS2322: Type '"green" | "white" | "orange" | "cyan" | "pink" | "purple" | "custom"' is not assignable to type '"green" | "white" | "orange" | "cyan" | "pink" | "purple" | undefined'.
  Type '"custom"' is not assignable to type '"green" | "white" | "orange" | "cyan" | "pink" | "purple" | undefined'.

2026-04-25T04:02:58.094209220Z [err]  sh build.sh
2026-04-25T04:02:58.112089828Z [err]  ERROR: failed to build: failed to solve: process "sh -c sh build.sh" did not complete successfully: exit code: 2

--- a/frontend/src/App.tsx
+++ b/frontend/src/App.tsx
@@ -1976,21 +1976,6 @@ export default function App() {
     link.href = '/logo.png'
   }, [])
 
-  const THEME_UNLOCK_LEVELS: Record<string, number> = { green: 1, white: 1, orange: 1, cyan: 3, pink: 5, purple: 7 }
-  const ALL_THEMES = ['green', 'white', 'orange', 'cyan', 'pink', 'purple', 'custom'] as const
-
-  const cycleTheme = () => {
-    const level = appCurrentLevel.level
-    const available = ALL_THEMES.filter(t => {
-      if (t === 'custom') return level >= 5 && customAccentColor !== ''
-      return (THEME_UNLOCK_LEVELS[t] || 1) <= level
-    })
-    const idx = available.indexOf(theme as any)
-    setTheme(available[(idx + 1) % available.length] as any)
-  }
-
-  const themeLabel = theme === 'custom' ? 'Custom' : theme === 'green' ? 'Green' : theme === 'white' ? 'White' : theme === 'orange' ? 'Orange' : theme === 'cyan' ? 'Cyan' : theme === 'pink' ? 'Pink' : 'Purple'
-  const themeDotColor = theme === 'green' ? 'bg-[#D7FE51]' : theme === 'white' ? 'bg-white' : theme === 'orange' ? 'bg-orange-400' : theme === 'cyan' ? 'bg-[#51FEFE]' : theme === 'pink' ? 'bg-[#FE51D7]' : theme === 'purple' ? 'bg-[#9B51FE]' : 'bg-[#00FF88]'
   const themeAccentHex = theme === 'custom' ? customAccentColor : theme === 'green' ? '#D7FE51' : theme === 'white' ? '#E5E7EB' : theme === 'orange' ? '#F97316' : theme === 'cyan' ? '#51FEFE' : theme === 'pink' ? '#FE51D7' : '#9B51FE'
 
   // Load users for admin
@@ -5146,7 +5131,7 @@ export default function App() {
           earnings={lootEarnings}
           ptoHours={lootPtoHours}
           durationMin={lootDurationMin}
-          theme={theme}
+          theme={theme === 'custom' ? undefined : theme}
         />
 
         {/* Shockwave ripple on clock in */}

